### ISSUE TITLE
Empty tag that is not self-closing generates invalid javascript

### ISSUE THREAD

**Issue by ruuda** [maciejhirsz/kobold#7]
The following snippet causes Kobold to generate invalid javascript:

```rust
html! {
    <svg></svg>
}
```
generates
```javascript
export function __e0_77bcc8fe2fa4f331() {
let e0=document.createElement("svg");
e0.append);
return e0;
}
```
which then causes
```
Uncaught SyntaxError: Unexpected token ')' (at inline0.js:3:10)
```

`<svg/>` and `<svg>""</svg>` both work fine, only an empty tag causes this problem.

**Comment by maciejhirsz** [comment]
Thanks for this, and sorry for being so slow, I was taking a bit of a coding break. Fixed and published in 0.4.1.

**Comment by ruuda** [comment]
Thanks for the fix!
